src/js/Setup.js

```diff
@@ -323,7 +323,14 @@ function createCindyNow() {
 function loadImage(obj) {
     var img;
     if (typeof obj === "string") {
-        img = new Image();
+        var ext = obj.split('.').pop().toLowerCase();
+        if (ext === "mp4" || ext === "webm" || ext === "ogg") {
+            img = document.createElement("video");
+            img.preload = "auto";
+            img.loop = true; //loop videos as default
+        } else {
+            img = new Image();
+        }
```

Comment:
I'm *slightly* unhappy with the fact that you base the decision on whether it's a video or a still image on the file extension. In the web file extensions are just a convention, not a requirement. MIME types are the thing that really counts. Which means that there may be quite a number of URLs out there which do not contain such an extension, or any extension at all, and which are still video.

Would it make sense to support `videos` as a parallel data structure to `images`, with the understanding that the former contains only videos, the latter only images, and if the creator of the widget doesn't know which one it is it's his responsibility to figure it out somehow? Internally it would all end up in the same collection of images.